partiql-ast/src/main/java/org/partiql/ast/v1/AstVisitor.java

```diff
@@ -149,7 +149,7 @@ public interface AstVisitor<R, C> {
 
     R visitSelectItem(SelectItem node, C ctx);
 
-    R visitSelectItemAll(SelectItem.Star node, C ctx);
```

Comment:
(self-review) I forgot to change naming of this visitor function after the class was renamed.